### DISCUSSION TITLE
Differences between small, medium and large models

### DISCUSSION THREAD

**Discussion by AlbertoZanatta** [explosion/spaCy#3517] [Help: Other Questions] [answered]
I know that the main differences between _sm, _md and _lg models are mostly statistical (the larger the model, the more accurate it is) and related to their respective size.
However, it is unclear to me why the 'large' models provide more reliable predictions. 
Is that because they use more features? Or maybe because they have been trained on a larger amount of data? Or maybe because of some pre-processing steps linked to the word vectors and vocabulary that they feature?
Can someone help me to shed some light on this issue?

**Comment by ines** [comment, marked as answer]
The central difference are the word vectors (and vocabulary). They're also the main reason for the difference in size – the rest of the components don't really change in size across models. 

If you provide word vectors during training, those will be used by the model, which can give you a nice boost in accuracy. However, it also means that the word vectors have to be present at runtime. If you look at the vectors info in the [models directory](https://spacy.io/models/en), you can see that the number of unique vectors differs between models. For example, the `md` model has 20k, and the regular `lg` model 685k.
